<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin</class>
 <widget class="QMainWindow" name="admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>847</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fumetteria</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QTabWidget" name="maintab">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="fumetti">
       <attribute name="title">
        <string>FUMETTI</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0" colspan="2">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Cerca:</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>NOME FUMETTO:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="searchLine"/>
               </item>
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>AUTORE:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="autoreLine"/>
               </item>
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>ARGOMENTO:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="genereLine"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Azioni:</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPushButton" name="getFumettoButton">
             <property name="text">
              <string>CHIEDI FUMETTO</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="addnewB">
             <property name="text">
              <string>AGGIUNGI FUMETTO</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="deleteButton">
             <property name="text">
              <string>CANCELLA FUMETTO</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>AGGIORNA IL DATABASE</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QTableView" name="tableView"/>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>Fumetto:</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="2" column="0">
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="0" column="2">
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="0" column="0">
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="0" column="1">
                <widget class="QLabel" name="coverLabel">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="pixmap">
                  <pixmap>img/No-Cover.png</pixmap>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <spacer name="horizontalSpacer_4">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item row="2" column="1">
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QLabel" name="labelIS">
                 <property name="text">
                  <string>Codice:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelName">
                 <property name="text">
                  <string>Titolo:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelDate">
                 <property name="text">
                  <string>Data pub.</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelA">
                 <property name="text">
                  <string>Autore:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelCP">
                 <property name="text">
                  <string>Copie disponibili:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelDesc">
                 <property name="text">
                  <string>Descrizione:</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="2" column="2">
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QLineEdit" name="lineIS"/>
               </item>
               <item>
                <widget class="QLineEdit" name="lineName"/>
               </item>
               <item>
                <widget class="QLineEdit" name="lineDate"/>
               </item>
               <item>
                <widget class="QLineEdit" name="lineA"/>
               </item>
               <item>
                <widget class="QLineEdit" name="lineCP"/>
               </item>
               <item>
                <widget class="QLineEdit" name="lineDesc"/>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="userfumetti">
       <attribute name="title">
        <string>MIEI FUMETTI</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="1" column="1">
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>Azioni:</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_10">
           <item row="1" column="0">
            <widget class="QPushButton" name="pushButton_2">
             <property name="text">
              <string>AGGIORNA DATABASE</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="returnButton">
             <property name="text">
              <string>RESTIUISCI FUMETTO</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QGroupBox" name="groupBox_6">
            <property name="title">
             <string>Fumetto:</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="0" column="1">
              <layout class="QGridLayout" name="gridLayout_9">
               <item row="0" column="0">
                <spacer name="horizontalSpacer_5">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="0" column="2">
                <spacer name="horizontalSpacer_6">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="0" column="1">
                <widget class="QLabel" name="coverLabelU">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="pixmap">
                  <pixmap resource="xml.qrc">:/img/No-Cover.png</pixmap>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <spacer name="horizontalSpacer_7">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item row="0" column="3">
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Codice:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Titolo:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>Data pub.</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>Autore:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>Descrizione:</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="0" column="4">
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <widget class="QLineEdit" name="lineISU"/>
               </item>
               <item>
                <widget class="QLineEdit" name="lineNameU"/>
               </item>
               <item>
                <widget class="QLineEdit" name="lineDateU"/>
               </item>
               <item>
                <widget class="QLineEdit" name="lineAU"/>
               </item>
               <item>
                <widget class="QLineEdit" name="lineDescU"/>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>Fumetti in mio possesso:</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QTableView" name="tableviewuser"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="utenti">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <attribute name="title">
        <string>UTENTI</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>811</width>
          <height>81</height>
         </rect>
        </property>
        <property name="title">
         <string>Cerca:</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_8">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>NOME UTENTE:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="sLineID"/>
           </item>
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>NOME:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="sLineName"/>
           </item>
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>COGNOME:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="sLineSname"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>400</y>
          <width>791</width>
          <height>61</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>ID Utente:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineIDU"/>
         </item>
         <item>
          <widget class="QPushButton" name="addUser">
           <property name="text">
            <string>AGGIUNGI UTENTE</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeUser">
           <property name="text">
            <string>TOGLI UTENTE</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupBox_8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>811</width>
          <height>301</height>
         </rect>
        </property>
        <property name="title">
         <string>Utenti:</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="0">
          <widget class="QTableView" name="tableUsersView"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>847</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAccount">
    <property name="title">
     <string>Account</string>
    </property>
    <addaction name="actionAccount_informations"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuAccount"/>
  </widget>
  <action name="actionAccount_informations">
   <property name="text">
    <string>Account Information</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="xml.qrc"/>
 </resources>
 <connections/>
</ui>
